### DISCUSSION TITLE
限定共有記事のシェア機能は警告があった方がいいかも

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#343] [要望・アイデア・不具合]
Qiitaの記事ページには、各種SNSへのシェアボタン（現在はTwitter、Facebook、はてなブックマークの3つ？）があります。
これは限定共有記事にも表示されており、クリックすると普通にシェアできるようになっています。
記事ページ上部にも注意メッセージは表示されていますが、URLの意図せぬ漏洩を防ぐためにも、限定共有記事をシェアしようとした場合改めて警告を表示したほうがいいかもしれません。

**Comment by tanaka-asami415** [comment]
@munierujp 
ご意見ありがとうございます🙏
限定共有記事にはシェアボタンを表示しないことも合わせて、検討させていただきます🙇
